src/OrchardCore.Modules/OrchardCore.Queries/Controllers/AdminController.cs

```diff
@@ -60,12 +65,6 @@ public async Task<IActionResult> Index(QueryIndexOptions options, PagerParameter
             var siteSettings = await _siteService.GetSiteSettingsAsync();
             var pager = new Pager(pagerParameters, siteSettings.PageSize);
 
-            // default options
-            if (options == null)
-            {
-                options = new QueryIndexOptions();
```

Comment:
doesn't need a new() ? I see the others did?

Reply:
It is the ContentOptions which is instanciated inside the QueriexIndexViewModel.